models/quality_measures/intermediate/nqf0420_pain_assessment_and_follow_up/quality_measures__int_nqf0420_denominator.sql

```diff
@@ -0,0 +1,201 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))) | as_bool
+   )
+}}
+
+with visit_codes as (
+
+    select
+          code
+        , code_system
+    from {{ ref('quality_measures__value_sets') }}
+    where lower(concept_name) in (
+          'home healthcare services'
+        , 'office visit'
+        , 'encounter to document medications'
+        , 'psychological and/or neuropsychological test administration'
+    )
+
+)
+
+, visits_encounters as (
+
+    select 
+          patient_id
+        , coalesce(encounter.encounter_start_date, encounter.encounter_end_date) as min_date
+        , coalesce(encounter.encounter_end_date, encounter.encounter_start_date) as max_date
+    from {{ ref('quality_measures__stg_core__encounter') }} encounter
+    inner join {{ ref('quality_measures__int_nqf0420__performance_period') }} as pp
+        on coalesce(encounter.encounter_end_date, encounter.encounter_start_date) >= pp.performance_period_begin
+            and coalesce(encounter.encounter_start_date, encounter.encounter_end_date) <= pp.performance_period_end
+    where lower(encounter_type) in (
+          'home health'
+        , 'office visit'
+        , 'outpatient'
+        , 'outpatient rehabilitation'
+    )
+
+)
+
+, procedures as (
+
+    select
+        patient_id
+      , procedure_date
+      , coalesce (
+              normalized_code_type
+            , case
+                when lower(source_code_type) = 'cpt' then 'hcpcs'
+                when lower(source_code_type) = 'snomed' then 'snomed-ct'
+                else lower(source_code_type)
+              end
+          ) as code_type
+        , coalesce (
+              normalized_code
+            , source_code
+          ) as code
+    from {{ ref('quality_measures__stg_core__procedure') }}
+    where   coalesce(modifier_1, '') not in ('GQ', 'GT', '95', 'POS 02')
+        and coalesce(modifier_2, '') not in ('GQ', 'GT', '95', 'POS 02')
+        and coalesce(modifier_3, '') not in ('GQ', 'GT', '95', 'POS 02')
+        and coalesce(modifier_4, '') not in ('GQ', 'GT', '95', 'POS 02')
+        and coalesce(modifier_5, '') not in ('GQ', 'GT', '95', 'POS 02')
```

Comment:
The last value, "POS 02," is not valid in this list. It refers to `place_of_service code = '02'`, which can be found in core.medical_claim. You can add this filter to the CTE below.